src/index.html

```diff
@@ -1,13 +1,677 @@
+              src="images/icons/quotes.svg"
+              alt="quotes"
+            >
+
+            <p class="testimonials-list__testimonial">
+              AIR raises the agency bar to stratospheric
+              heights on both creative output and client service.
+            </p>
+
+            <p class="testimonials-list__name">Michel Grover</p>
+
+            <h5 class="testimonials-list__profession section-title--h5">Hulu</h5>
+          </div>
+        </div>
+      </section>
+
+      <section class="vision-passion-results">
+        <div class="vision-passion-results__wrapper">
+          <h2
+            class="vision-passion-results__title
+            section-title
+            section-title--h2"
+          >
+            Vision, Passion, Results
+          </h2>
+
+          <p class="vision-passion-results__description">
+            We are sure that first-rate job is possible
+            only if all three components are united.
+          </p>
+
+          <a href="#contact-us" class="vision-passion-results__button-link">
+            <button
+              class="button"
+              type="submit"
+            >
+              Apply
+            </button>
+          </a>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="footer__wrapper">
+        <section class="send-us-a-message" id="send-us-a-message">
+          <h3 class="send-us-a-message__title section-title--h3">
+            Send us a message
+          </h3>
+
+          <form
+            class="send-us-a-message__input input"
+            action="#"
+            method="post"
+            onsubmit="this.reset(); return false;"
+          >
+            <input
+              class="input__fields"
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Your email"
+              required
+            >
+
+            <input
+              class="input__fields"
+              type="text"
+              name="name"
+              id="name"
+              placeholder="Your name"
+              required
+            >
+
+            <textarea
+              class="input__fields input__fields--textarea"
+              name="comments"
+              id="textarea"
+              cols="30"
+              rows="1"
+              placeholder="Your message"
+              required
+              minlength="30"
+            ></textarea>
+
+            <button
+              class="button button--send"
+              type="submit"
+            >
+              Send
+            </button>
+          </form>
+        </section>
+
+        <address class="contact-us" id="contact-us">
+          <h3 class="contact-us__title section-title--h3">
+            Contact us
+          </h3>
+
+          <div class="contact-us__phone-container">
+            <h5 class="contact-us__call-us">Call us</h5>
+
+            <a class="contact-us__phone-link" href="tel:654 321 987">
```

Comment:
Need to fix format phone only in href
```suggestion
            <a class="contact-us__phone-link" href="tel:654321987">
```